shifthelper/tools/__init__.py

```diff
@@ -128,5 +128,5 @@ def retry_fetch_fail_after_30sec():
         ).json()
     try:
         return retry_fetch_fail_after_30sec()
-    except RetryError as e:
```

Comment:
this "as e" could be reverted I assume

Reply:
This was removed because pyflakes complaint about an unused variable